app/main.py

```diff
@@ -2,0 +1,49 @@
+from typing import List
+
+
+class Animal:
+    alive: List["Animal"] = []
+
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name: str = name
+        self.appetite: int = appetite
+        self.is_hungry: bool = is_hungry
+        Animal.alive.append(self)
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
```

Comment:
This function could be a static method, because it doesn't use any instance variables or methods. In Python, we can use the `@staticmethod` decorator to indicate that a method doesn't depend on any instance variables or methods. It's a good practice to use static methods when possible, because it makes the code more modular and easier to test.